developer_guides/rimage/ipc4_extended_module_config.rst

```diff
@@ -0,0 +1,219 @@
+.. _ipc4_extended_module_config:
+
+Ipc4_extended_module_config
+###########################
+
+The ipc4 extended module config is stored in the extended manifest structure of the
+output binary file. This config includes many module attributes such as name, uuid
+or capabilities. The host driver will rely on the extended module config for IPC
+messages to the DSP
+
+Rimage builds extended module config based on module setting in config/xxx.toml.
+The following setting is for gain module.
+
+.. code-block:: bash
+
+        [[module.entry]]
+        name = "GAIN"
+        uuid = "61BCA9A8-18D0-4A18-8E7B-2639219804B7"
+        affinity_mask = "0x1"
+        instance_count = "40"
+        domain_types = "0"
+        load_type = "0"
+        module_type = "5"
```

Comment:
@RanderWang while this is still pending, can you please also add the init_config part that I recently introduced?

Reply:
sure.